src/qt/votingdialog.cpp

```diff
@@ -762,15 +822,297 @@ VotingVoteDialog::VotingVoteDialog(QWidget *parent)
     answer_->setAlignment(Qt::AlignLeft|Qt::AlignVCenter);
     answer_->setTextInteractionFlags(Qt::TextSelectableByMouse);
     glayout->addWidget(answer_, 3, 1);
+    
+    answerList_ = new QListWidget(this);
+    vlayout->addWidget(answerList_);
+    
+    QHBoxLayout *hlayout = new QHBoxLayout();
+    vlayout->addLayout(hlayout);
+    
+    QPushButton *voteButton = new QPushButton();
+    voteButton->setText(tr("Vote"));
+    hlayout->addWidget(voteButton);
+    connect(voteButton, SIGNAL(clicked()), this, SLOT(vote()));
+    
+    voteNote_ = new QLabel();
+    voteNote_->setAlignment(Qt::AlignLeft|Qt::AlignVCenter);
+    voteNote_->setTextInteractionFlags(Qt::TextSelectableByMouse);
+    voteNote_->setWordWrap(true);
+    hlayout->addWidget(voteNote_);
+    
+    QString sVoteTitle;
 }
 
 void VotingVoteDialog::resetData(const VotingItem *item)
 {
     if (!item)
         return;
+    
+    answerList_->clear();
+    voteNote_->clear();
+    question_->setText(item->question_);
+    url_->setText(item->url_);
+    answer_->setText(item->bestAnswer_);
+    sVoteTitle=item->title_;
+    std::string listOfAnswers = item->answers_.toUtf8().constData();
+    std::vector<std::string> vAnswers = split(listOfAnswers, ";");
+    for(size_t y=0; y < vAnswers.size(); y++) {
+        QListWidgetItem *answerItem = new QListWidgetItem(QString::fromStdString(vAnswers[y]),answerList_);
+        answerItem->setCheckState(Qt::Unchecked);
+    }
+}
 
-    question_->setText(QString(tr("Q: ")) + item->question_);
-    url_->setText(QString(tr("Discussion URL: ")) + item->url_);
-    answer_->setText(QString(tr("Best Answer: ")) + item->bestAnswer_);
+void VotingVoteDialog::vote(void)
+{
+    QString sVoteValue = GetVoteValue();
+    voteNote_->setStyleSheet("QLabel { color : red; }");
+    
+    if(sVoteValue.isEmpty()){
+        voteNote_->setText(tr("Vote failed! Select one or more items to vote."));
+        return;
+    }
+    
+    // replace spaces with underscores
+    sVoteValue.replace(" ","_");
+    sVoteTitle.replace(" ","_");
+    
+    std::string sResult = ExecuteRPCCommand("vote", sVoteTitle.toStdString(), sVoteValue.toStdString());
+    
+    if (sResult.find("Success") != std::string::npos) {
+        voteNote_->setStyleSheet("QLabel { color : green; }");
+    }
+    voteNote_->setText(QString::fromStdString(sResult)); 
+}
+
+QString VotingVoteDialog::GetVoteValue(void)
+{
+    QString sVote = "";
+    for(int row = 0; row < answerList_->count(); row++) {
+        QListWidgetItem *item = answerList_->item(row);
+        if(item->checkState() == Qt::Checked)
+            sVote += item->text() + ";";
+    }
+    sVote.chop(1);
+    return sVote;
+}
+
+// NewPollDialog
+//
+NewPollDialog::NewPollDialog(QWidget *parent)
+    : QDialog(parent)
+{
+    setWindowTitle(tr("Create Poll"));
```

Comment:
Doi you mean .ui files for qt designer? I don't know. The votingdialog doesn't use .ui layout files, the rest of the wallet does more or less.